### PR TITLE
Adapting ExchangeRatesApi to the new API format

### PR DESCRIPTION
**Pull request by Seagull-4auKa**
Split endpoints by `enterprise` option
Handle new response format
The issue is described here: https://github.com/florianv/exchanger/issues/135

For discussion:
https://github.com/Seagull-4auKa/exchanger/blob/0e93e602455feac0566a76afef2c58469128255e/src/Service/ExchangeRatesApi.php#L37-L39
I didn't rename const `LATEST_URL` and `HISTORICAL_URL` to `ENTERPRISE_LATEST_URL` and `ENTERPRISE_HISTORICAL_URL` respectively. I don't think they can be used outside of the class or tests, but still left them as they are.

### CODE REVIEW

File: src/Service/ExchangeRatesApi.php [hunk 1]
```diff
@@ -90,16 +113,13 @@ protected function getHistoricalExchangeRate(HistoricalExchangeRateQuery $exchan
      */
     public function supportQuery(ExchangeRateQuery $exchangeQuery): bool
     {
-        return true;
+        return $this->options['enterprise'] || 'EUR' === $exchangeQuery->getCurrencyPair()->getBaseCurrency();
```
Comment (karser) [inline src/Service/ExchangeRatesApi.php:116]: Why EUR? It's USD, isnt it?

exchangeratesapi | USD (free), * (paid)
-- | --

Comment (Seagull-4auKa) [reply src/Service/ExchangeRatesApi.php:116]: My mistake is not changing the readme.

I'll commit today

https://exchangeratesapi.io/documentation/#api_response
```
The delivered exchange rates with the API, are by default connected to the Euro
```

**Comment by karser** [comment]
For me, your PR works fine, thanks!

**Comment by Seagull-4auKa** [comment]
Merge with master
Fix ExchangeRatesApiTest.php method names to snake_case
Update doc/readme.md
Update README.md - update base currency info for ExchangeRatesApi

**Comment by florianv** [comment]
Thanks @Seagull-4auKa :)

**Pull request merged**